### ISSUE TITLE
Logical XOR

### ISSUE THREAD

**Issue by pxeger** [cairdcoinheringaahing/jellylanguage#1]
```
logical_xor(zero, zero) = false
logical_xor(zero, non_zero) = true
logical_xor(non_zero, zero) = true
logical_xor(non_zero, non_zero) = false
```

This is based on whether the argument is truthy or falsey, not anything to do with its bits.

Also `xnor`, which is `not(logical_xor(a, b))`

**Comment by cairdcoinheringaahing** [comment]
I don't see this coming up enough to dedicate a single byte dyad to, however a digraph (e.g. `œx` as `œ^` already exists) could certainly be useful. As for `xnor`, I'm not sure if it'd be worth it over just `œx¬`

**Comment by pxeger** [comment]
I agree, a single byte would certainly be wasteful. And you're probably right about xnor.
